<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombatWindow</class>
 <widget class="QWidget" name="CombatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>전투창</string>
  </property>
  <widget class="QProgressBar" name="bar_playerhp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>118</width>
     <height>10</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar_monsterhp">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>16</y>
     <width>118</width>
     <height>10</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>236</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_playerhp">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_player">
      <property name="text">
       <string>Player</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>77</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_CombatAttack">
      <property name="text">
       <string>일반 공격</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_SpecialAttack">
      <property name="text">
       <string>특수 공격</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_CombatFlee">
      <property name="text">
       <string>도망</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>13</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_monster">
      <property name="text">
       <string>Monster</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_monsterhp">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="txtCombatLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>77</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutSkill">
    <item>
     <widget class="QPushButton" name="btn_CombatSkill1">
      <property name="text">
       <string>스킬1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_CombatSkill2">
      <property name="text">
       <string>스킬2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_CombatSkill3">
      <property name="text">
       <string>스킬3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
